lib/std/time/Date.zig

```diff
@@ -0,0 +1,265 @@
+year: Year = 1970,
+month: Month = .January,
+day_of_month: DayOfMonth = 1, // 1 - 31
+
+const Date = @This();
+const std = @import("../std.zig");
+const s_per_day = std.time.s_per_day;
+
+const seconds_from_epoch_to_y2k = 30 * (365 * s_per_day) + 7 * s_per_day;
+
+pub const Year = i32;
```

Comment:
Some of those are also defined in https://github.com/ziglang/zig/blob/master/lib/std/time/epoch.zig#L69, it would be nice if they could be consolidated into a single place. (Perhaps have the constants in `epoch.zig` use `time.Month` etc.

Reply:
I'm aware, but I did not touch epoch.zig in the interest of not breaking anything.  I will create a follow up issue when this is merged.